javascript/src/components/dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,60 @@
+/* global jsRoutes */
+
+const React = require('react');
+const DropdownButton = require('react-bootstrap').DropdownButton;
+const MenuItem = require('react-bootstrap').MenuItem;
+
+const EditDashboardModalTrigger = require('./EditDashboardModalTrigger');
+const PermissionsMixin = require('../../util/PermissionsMixin');
+
+const DashboardStore = require('../../stores/dashboard/DashboardStore');
```

Comment:
Could you please use [module imports](https://github.com/airbnb/javascript#10.1)? That way imports are theoretically independent of the module loader user.